### PR TITLE
Remove DOCS.md and link to ReadTheDocs in README

### PR DESCRIPTION
**Pull request by ghost**
To ensure that we don't have to maintain duplicate information, I've added a link to ReadTheDocs. I've also added a badge because I like badges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ Pass a comma-separated list of modules to run a subset of the plugins:
 $ botbot-shell brain,images
 ```
 
+## Docs
+[![Read the Docs](https://img.shields.io/readthedocs/pip.svg)](https://brainzbot.readthedocs.io/)
+
+BrainzBot's documentation, including information on how to create new plugins, is available on [ReadTheDocs](https://brainzbot.readthedocs.io).
```
Comment (LeoVerto) [inline README.md]: The official spelling seems to be "Read the Docs".

File: README.md [hunk 2]
```diff
@@ -13,6 +13,11 @@ Pass a comma-separated list of modules to run a subset of the plugins:
 $ botbot-shell brain,images
 ```
 
+## Docs
+[![Read the Docs](https://img.shields.io/readthedocs/pip.svg)](https://brainzbot.readthedocs.io/)
```
Comment (LeoVerto) [inline README.md:17]: Love the badge!

**Review by LeoVerto — CHANGES_REQUESTED** [review]
Fix the "Read the Docs" spelling and this is good to go!

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Sorry, amended!

**Pull request merged**